From weiyi.huang@gmail.com  Mon Oct 20 13:58:20 2008
From: Huang Weiyi <weiyi.huang@gmail.com>
Date: Sat, 18 Oct 2008 11:53:05 +0800
Subject: Staging: wlan-ng: prism2: remove duplicated #include
To: gregkh@suse.de
Cc: linux-kernel@vger.kernel.org
Message-ID: <20081018114433.61C0.WEIYI.HUANG@gmail.com>


Removed duplicated #include "wlan_compat.h" in
drivers/staging/wlan-ng/prism2mib.c.

Signed-off-by: Huang Weiyi <weiyi.huang@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/wlan-ng/prism2mib.c |    2 --
 1 file changed, 2 deletions(-)

--- a/drivers/staging/wlan-ng/prism2mib.c
+++ b/drivers/staging/wlan-ng/prism2mib.c
@@ -90,8 +90,6 @@
 #include <linux/usb.h>
 //#endif
 
-#include "wlan_compat.h"
-
 /*================================================================*/
 /* Project Includes */
 
